client/src/pages/BusinessDetails.js

```diff
@@ -1,89 +1,246 @@
+        // console.log(bizService)
+        dbServices?.services.find(service => {
+          if (service.id === bizService) {
+            tempBizServiceArr.push(service)
+          }
+        })
+      })
+
+      setBizServiceArr(tempBizServiceArr)
+      // console.log('bizServiceArr', bizServiceArr)
+    }
+    fetchDetails()
+    return function clean() {
+      setIsLoading(false)
+    }
+  }, [business])
+
   return (
-    <React.Fragment>
-      <ContentHeader title='Business Details ' />
-      <div className='card w-50 mx-auto'>
-        <div className='card-body'>
-          <ul className='products-list product-list-in-card'>
-            <li className='item'>
-              <figure className='indie-card'>
-                <div className='pic-star-container'>
-                  <img
-                    src='https://via.placeholder.com/100'
-                    alt='Placeholder Business Profile'
-                    className='image-fluid'
-                  />
-                  <p className='bus-details--rating '>
-                    <img
-                      src='star.svg'
-                      alt='Star Icon'
-                      style={{ width: '10%' }}
-                    />
-                    <img
-                      src='star.svg'
-                      alt='Star Icon'
-                      style={{ width: '10%' }}
-                    />
-                    <img
-                      src='star.svg'
-                      alt='Star Icon'
-                      style={{ width: '10%' }}
-                    />
-                    <img
-                      src='star.svg'
-                      alt='Star Icon'
-                      style={{ width: '10%' }}
-                    />
-                  </p>
-                  {props.token ? (
-                    <button className='btn btn-default'>Review</button>
-                  ) : (
-                    ''
-                  )}
-                </div>
-                <div>
-                  <h1 className='product title'>{business.businessName}</h1>
-                  <div>
-                    {business.address1} {business.city}, {business.state}{' '}
-                    {business.zipCode}
-                    <br />
-                    email: {business.email}
-                    <div className={phone ? 'visable-phone' : 'hidden-phone'}>
-                      phone: {business.phone}
+    <>
+      {!props.loading ? (
+        <>
+          {/* (Backlog) TODO: [ ] - ? Remove all ContentHeaders or modify and utilize across app   */}
+          {/* <ContentHeader title='Business Details ' /> */}
+          <section className='content'>
+            <div className='card mx-auto'>
+              <div className='card-body'>
+                <div className='products-list product-list-in-card d-flex justify-content-around align-content-center'>
+                  <div className='m-3'>
+                    <figure className='bus-details--image'>
```

Comment:
also deleted 102 - 105  and simplified CSS (106, 107) but didn't have your info yet..